lib/neat/version.rb

```diff
@@ -1,3 +1,3 @@
 module Neat
-  VERSION = "1.7.4"
+  VERSION = "2.0.0"
```

Comment:
Should I do something about these failures? I didn't see a editor file that controlled the ruby style guide. If there isn't a solution already I could commit a rubocop file if you'd like.